docs/source/install.rst

```diff
@@ -56,24 +56,33 @@ Installing Graphviz
 In order to use :meth:`EntitySet.plot <featuretools.entityset.EntitySet.plot>` or :func:`featuretools.graph_feature`
 you will need to install the graphviz library.
 
-Conda users::
+pip users::
 
-    conda install python-graphviz
+    pip install graphviz
+    
+conda users::
+
+    conda install -c conda-forge python-graphviz
 
 Ubuntu::
 
-    sudo apt-get install graphviz
+    sudo apt install graphviz
     pip install graphviz
 
 Mac OS::
 
     brew install graphviz
     pip install graphviz
 
-Windows::
+Windows:
 
-    conda install python-graphviz
+- Install the graphviz from the `official source <https://graphviz.org/download/#windows>`_
```

Comment:
they only need to install graphviz from the official site if they use the pip install

Reply:
Fixed